### PR TITLE
Show error when user removes the last secondary authenticator if it is required

### PR DESCRIPTION
**Pull request by tung2744**
ref DEV-2249

Screenshot 2024-10-31 at 4 00 04 PM

### CODE REVIEW

File: resources/authgear/templates/en/translation.json [hunk 1]
```diff
@@ -942,6 +942,7 @@
   "v2.component.verify-bot-protection.default.title": "Checking your system...",
   "v2.error.account-conflict": "{ IdentityTypeIncoming, select, login_id {{FlowType, select, login {Looks like you used {OAuthProviderNameExisting} for login before.} other {This {LoginIDNameIncoming} is already registered.}}} oauth {Looks like you haven''t linked up with that {OAuthProviderNameIncoming} account yet.} other {This account is already registered} }",
   "v2.error.account-not-found": "{ IdentityTypeIncoming, select, login_id {{ LoginIDTypeIncoming, select, email {This email} phone {This phone number} username {This username} other {This identity} }} oauth {This email} other {This identity} } cannot be found. Do you want to create an account?",
+  "v2.error.secondary-authenticator-required": "At least one 2-factor authentication method is required, please add another method before removing this one.",
```
Comment (louischan-oursky) [inline resources/authgear/templates/en/translation.json]: We have v2.error.remove-last-secondary-authenticator already. Let's reuse that?

**Review by louischan-oursky — CHANGES_REQUESTED** [review]

File: resources/authgear/templates/en/translation.json (hunk 1 above)
Comment (tung2744) [reply resources/authgear/templates/en/translation.json]: Oh doesn't know that. Maybe I should update the reason to `RemoveLastSecondaryAuthenticator` too.

Comment (louischan-oursky) [reply resources/authgear/templates/en/translation.json]: No. Use the new error please.

**Comment by tung2744** [comment]
Updated to reuse existing key, thanks!

**Review by louischan-oursky — APPROVED** [review]

**Pull request merged**